idunn/blocks/wikipedia.py

```diff
@@ -3,11 +3,14 @@
 from apistar import validators
 from .base import BaseBlock, BlocksValidator
 from requests.exceptions import HTTPError, RequestException, Timeout
+import pybreaker
+
+wiki_breaker = pybreaker.CircuitBreaker(fail_max=15, reset_timeout=120)
 
 def handle_requests_error(f):
     def wrapper(*args, **kwargs):
         try:
-            return f(*args, **kwargs)
+            return wiki_breaker.call(f,*args, **kwargs)
```

Comment:
I don't understand the kind of error you catch with the circuit breaker.
we don't want to activate it for 404 or 400.

I think we want it only for 500 and timeout